radionice/pocetni/lekcije/003_imenice_i_pokazne_zamjenice/003_imenice_i_pokazne_zamjenice.tex

```diff
@@ -121,12 +123,12 @@
 		\rei{おかあさん　の　ともだち　の　ねこ}{mačka mamine prijateljice}
 	\end{reibun}
 	
-	Ovdje se nalaze dva ulančana opisa:  \textit{ともだち　の　ねこ prijateljičina mačka} i \textit{おかあさん　の　ともだち majčina prijateljica}.
-	
-	\begin{reibun}
-		\rei{たなか　さん　の　ほん　の　おもさ}{težina Tanakine knjige}
-		\rei{この　いえ　の　たかさ}{visina ove kuće}
-	\end{reibun}
+	Ovdje se nalaze dva ulančana opisa: 
+        \begin{reibun}
+                \rei{ともだち　の　ねこ}{prijateljičina mačka}
+                \rei{おかあさん　の　ともだち}{majčina prijateljica}
+        \end{reibun}
```

Comment:
Poanta ovih primjera je bila da imaju 3 imenice u odnosu. Promjena je dobra što se tiče izbora riječi, ali trebali bismo dodati i treću imenicu, npr. da bude _boja prijateljičine mačke_.

Reply:
Stavila sam da bude boja prijateljičine mačke kako je predlozeno. Nadam se i da sam to napisala na japanskom :)

Reply:
Jesi :)